Refactor Kiali documentation (#8461)
kyma-project/kyma: docs/kiali/01-01-kiali.md

---
title: Overview
---

Kyma uses [Kiali](https://www.kiali.io) to enable validation, observe the Istio Service Mesh, and provide details on microservices included in the Service Mesh and connections between them.

> Kiali offers a set of dashboards and graphs that allow to have the full service mesh at a glance and quickly identify problems and configuration issues.

Reviewer: ```suggestion
Kiali offers a set of dashboards and graphs that allow you to have the full Service Mesh at a glance and quickly identify problems and configuration issues.
```

> For a more detailed feature documentation, please refer to the [official kiali documentation](https://kiali.io/documentation/features/).

Reviewer: ```suggestion
For more details about particular features, see the [official Kiali documentation](https://kiali.io/documentation/features/).
```

> 

Reviewer: Remove the redundant line :)

Author: Done thanks!

> ---

Reviewer: As Kiali is the new component, we need to define a structure on the website. I can do this later :)

title: Overview
---

Kyma uses [Kiali](https://www.kiali.io) to enable validation, observe the Istio Service Mesh, and provide details on microservices included in the Service Mesh and connections between them.
Kiali offers a set of dashboards and graphs that allow to have the full service mesh at a glance and quickly identify problems and configuration issues.

> Some of its features are:

Reviewer 2: That list of features is changing and some are even not part of the default setup as for example the Istio wizards or the custom dashboards, or the tracing integration.
I prefer to not list it here but just have the link to the upstream docu

Author: Removed list and linked equivalent page in kiali docs.

- Observability Features
    - Graphs
        - Health
        - Drill-Down
        - Side-Panel
        - Traffic Animation
        - Graph Types
    - Detail Views
        - Metrics
        - Services
        - Workloads
        - Custom Dashboards
    - Distributed Tracing
- Configuration and Validation Features
    - Istio Configuration
    - Validations
    - Istio Wizards
        - Weighted Routing Wizard
        - Matching Routing Wizard
        - Suspend Traffic Wizard
        - Advanced Options
        - More Wizard examples

For a more detailed feature documentation, please refer to the [official kiali documentation](https://kiali.io/documentation/features/).

> >**NOTE:** Kiali is disabled by default. Read [this](/root/kyma/#configuration-custom-component-installation) document for instructions on how to enable it.

Reviewer 2: It is disabled in "Kyma Lite" aka minikube setup, but in the regular installation it is included

Author: Clarified that this is only valid for kyma lite (local setup).

---
title: Overview
---

Kyma uses [Kiali](https://www.kiali.io) to enable validation, observe the Istio Service Mesh, and provide details on microservices included in the Service Mesh and connections between them.
Kiali offers a set of dashboards and graphs that allow you to have the full Service Mesh at a glance and quickly identify problems and configuration issues.
For more details about particular features, see the [official Kiali documentation](https://kiali.io/documentation/features/).

> >**NOTE:** Kiali is disabled by default in Kyma Lite (local installation). Read [this](/root/kyma/#configuration-custom-component-installation) document for instructions on how to enable it.

Reviewer: ```suggestion
>**NOTE:** Kiali is disabled by default in Kyma Lite (local installation). Read about [custom component installation](/root/kyma/#configuration-custom-component-installation) for instructions on how to enable it.
```
